internal/rsmq/rsmq.go

```diff
@@ -2,11 +2,14 @@ package rsmq
 
 import (
 	"errors"
+	"fmt"
 	"strconv"
 	"strings"
 	"time"
 
 	"github.com/go-redis/redis"
```

Comment:
hmm i'm not 100% sure why we used the "old redis" client? will need to look more into this. Maybe there was some issue or it's just a mistake and should be upgraded.

Reply:
From the description:

> RSMQ Compatibility
> 
> - Scheduler service maintains separate Redis client for RSMQ library compatibility
> - Uses legacy Redis package (github.com/go-redis/redis) for RSMQ operations
> - Automatic cluster/single-node client selection for RSMQ

I believe it may be possible to migrate this. But it felt like this was already quite a big change.